core/src/wsv.rs

```diff
@@ -190,7 +190,7 @@ impl WorldStateView {
     ///
     /// Return a Boolean value indicating whether or not the  [`Account`] already had this permission.
     pub fn add_account_permission(
-        &self,
+        &mut self,
```

Comment:
Yea, I didn't want to change the implementation in case I broke something. But I have a short term plan for how we can remove interior mutability. It's not that far.